.github/workflows/ci.yml

```
@@ -199,8 +202,11 @@ jobs:
       - name: Take snapshot of installed and running apps and services
         id: snapshot
         run: |
-          brew ruby -r "$(brew --repository homebrew/cask)/cmd/lib/check.rb" <<'EOF'
-            puts "::set-output name=before::#{JSON.generate(Check.all)}"
+          brew ruby -r "$(brew --repository homebrew/cask)/cmd/lib/check.rb" <<'EOF' -- $GITHUB_OUTPUT
+            github_output, = ARGV
+            File.open(github_output, "a") do |f|
+              f.puts "before=#{JSON.generate(Check.all)}"
+            end
```

Stuff like this is probably actually simpler if we just:
```suggestion
          brew ruby -r "$(brew --repository homebrew/cask)/cmd/lib/check.rb" >> $GITHUB_OUTPUT <<'EOF' 
            puts "before=#{JSON.generate(Check.all)}"
          end
```

---

It can, but if currently are not then there's nothing to worry about.

If we are: making it stderr instead will fix the issue.